docs/how-to/modify-kernel-boot-parameters.rst

```diff
@@ -57,6 +57,14 @@ Append parameters to the line that starts with ``linux``:
 * Don't add space round ``=`` and other punctuations for each key-value
 * Don't add line breaks
 
+For example, after adding ``nohz=on`` and ``nohz_full=1-3``:
+
+.. code-block:: text
+
+           linux        /boot/vmlinuz-5.15.0-1056-realtime root=UUID=06cfb4\
+   f1-21f9-4568-b656-926224e20fb3 ro quiet splash $vt_handoff nohz=on nohz_\
+   full=1-3
```

Comment:
I believe the slashes are to indicate visual word wrappings, they should not be part of the value.
```suggestion
           linux        /boot/vmlinuz-5.15.0-1056-realtime root=UUID=06cfb4f1-21f9-4568-b656-926224e20fb3 ro quiet splash $vt_handoff nohz=on nohz_full=1-3
```

Reply:
yup the back slashes are for the line wraps; but I thought it might be clearer if it's included when compared to the screenshot.
but i can remove it; probably less confusing.